services/minespace-web/common/actionCreators/projectActionCreator.js

```diff
@@ -132,12 +132,43 @@ export const fetchProjectSummaries = (payload) => (dispatch) => {
     .get(ENVIRONMENT.apiUrl + API.PROJECT_PROJECT_SUMMARIES(payload), createRequestHeader())
     .then((response) => {
       dispatch(success(reducerTypes.GET_PROJECT_SUMMARIES));
-      dispatch(projectSummaryActions.storeProjectSummaries(response.data));
+      dispatch(projectActions.storeProjectSummaries(response.data));
     })
     .catch(() => dispatch(error(reducerTypes.GET_PROJECT_SUMMARIES)))
     .finally(() => dispatch(hideLoading()));
 };
 
+export const fetchProjectsByMine = ({ mineGuid }) => (dispatch) => {
+  dispatch(request(reducerTypes.GET_PROJECTS));
+  dispatch(showLoading());
+  return CustomAxios({ errorToastMessage: Strings.ERROR })
+    .get(ENVIRONMENT.apiUrl + API.PROJECTS(mineGuid), createRequestHeader())
+    .then((response) => {
+      dispatch(success(reducerTypes.GET_PROJECTS));
+      dispatch(projectActions.storeProjects(response.data));
+    })
+    .catch(() => dispatch(error(reducerTypes.GET_PROJECTS)))
+    .finally(() => dispatch(hideLoading()));
+};
+
+export const fetchProjectById = (projectGuid) => (dispatch) => {
+  dispatch(request(reducerTypes.GET_PROJECT));
+  dispatch(showLoading());
+  return CustomAxios({ errorToastMessage: Strings.ERROR })
+    .get(ENVIRONMENT.apiUrl + API.PROJECT(projectGuid), createRequestHeader())
+    .then((response) => {
+      dispatch(success(reducerTypes.GET_PROJECT));
+      dispatch(projectActions.storeProject(response.data));
+      dispatch(success(reducerTypes.GET_PROJECT_SUMMARY));
```

Comment:
So far, I have not found any comments. Just curious on this one:
Why this is not added Project summary fetchProjectsByMine as in here?

Reply:
Thanks for clarifying :)